config-templates/redhat/odf-local/4.10/deployment.yaml

```diff
@@ -467,6 +470,7 @@ items:
       hpcsBaseUrl: "{{{ kms-base-url }}}"
       hpcsTokenUrl: "{{{ kms-token-url }}}"
       {{/kms-encryption}}
+      ignoreNoobaa: "{{ ignore-noobaa }}"
```

Comment:
since `ignore-noobaa` is of `boolean` type, remove  double quotes from `"{{ ignore-noobaa }}"`